jupyterhub/templates/hub/deployment.yaml

```diff
@@ -47,9 +47,9 @@ spec:
         {{- .Values.hub.extraVolumes | toYaml | trimSuffix "\n" | nindent 8 }}
         {{- end }}
         {{- if eq .Values.hub.db.type "sqlite-pvc" }}
-        - name: hub-db-dir
+        - name: pvc
```

Comment:
This is a possibly breaking change, btw. If you are using initContainers that also mount `/srv/jupyterhub`, you'll end up with this error:

```
Error: UPGRADE FAILED: cannot patch "hub" with kind Deployment: Deployment.apps "hub" is invalid: spec.template.spec.containers[1].volumeMounts[0].name: Not found: "hub-db-dir"
```

We should at least note this somewhere.

Reply:
Ah good point